src/fetch_agent/fetch_agent/agent.py

```diff
@@ -43,24 +58,102 @@ async def startup_event(ctx: Context):
     minimal_publisher.log_message(f"Agent: {agent.name} ({agent.address})")
 
     # run function in background so agent can fully start while registering
-    asyncio.ensure_future(register_at_registry(ctx, agent))
-
-    queuing_system = QueuingSystem(reservations=[])
+    asyncio.ensure_future(register_at_registry(ctx))
 
     properties = PropertyData(
-        open_time_frames=queuing_system.open_time_frames(),
+        open_time_frames=[(0, 0), (0, 0)],
         geo_point=(20.32, 85.52),
         cost_per_kwh=34.76,
         charging_wattage=11,
         green_energy=False,
     )
 
```

Comment:
```suggestion

ctx.storage.set("queuing_system", queuing_system.to_json())
```

should also be reverted